src/v2/Components/ArtworkFilter/ArtworkFilters/index.tsx

```diff
@@ -21,6 +26,24 @@ interface ArtworkFiltersProps {
 // Some filters will be rendered only if there is the necessary data in aggregations (for example, ArtistsFilter)
 export const ArtworkFilters: React.FC<ArtworkFiltersProps> = props => {
   const { user, relayEnvironment } = props
+  const {
+    contextPageOwnerId,
+    contextPageOwnerSlug,
+    contextPageOwnerType,
+  } = useAnalyticsContext()
+
+  const variant = useFeatureVariant("filters-expanded-experiment")
+
+  useTrackVariantView({
```

Comment:
**question**: I feel like this hook needs to be deconstructed into functions that can be called conditionally or it needs to support a prop that signals whether the analytics call should be sent.

As is, we can't supply any conditions here which is tricky when we want to use the hook within shared components.

@gkartalis Maybe for now we can call `trackEvent` explicitly and not rely on this new hook?

Reply:
Will call `trackEvent` explicitly for now 💯